kubernetes/istio.go

```diff
@@ -215,7 +212,7 @@ func (in *K8SClient) getIstiodDebugStatus(debugPath string) (map[string][]byte,
 
 	// Check if the kube-api has proxy access to pods in the istio-system
 	// https://github.com/kiali/kiali/issues/3494#issuecomment-772486224
-	_, err = in.GetPodProxy(c.IstioNamespace, istiods[0].Name, "/ready")
+	_, err = in.ForwardGetRequest(c.IstioNamespace, istiods[0].Name, httputil.GetFreePort(), 8080, "/ready")
```

Comment:
Code suggestion.
It would be nice if we add a note on the destination port, what is used, etc.
That may help to identify in the future if there is a change, and also I'm just thinking for us.
For example, we've reviewed that the 8080 would be deprecated for one type of service but it's still used for readiness.